phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -3153,6 +3158,7 @@ public boolean isViewReferenced() {
             } else {
                 tableUpsert.setNull(36, Types.VARCHAR);
             }
+            tableUpsert.setLong(37, maxLookbackAge);
```

Comment:
When creating a table from now on maxlookbackage will default to 0 instead of NULL? Are we sure that we aren't introducing assumptions that max lookback age is non null elsewhere, since existing tables will have the value be null?

Reply:
Got it, so I think the max lookback age as undefined should be either null or 0. And, preferrable will be null as already for existing tables we will have it as NULL. Please correct me if I am understanding it wrong.

Reply:
Actually I was of the understanding that I took care of converting existing NULLs in SYSCAT to 0 (default) via: https://github.com/sanjeet006py/phoenix/blob/PHOENIX-7006/phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java#L1448, but I agree it good to be consistent around definition of max lookback age not defined.